cinema/urls.py

```diff
@@ -1,10 +1,28 @@
 from django.urls import path
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import genre_list, genre_detail, ActorList, ActorDetail, CinemaHallDetail, CinemaHallList, MovieViewSet
+
+movie_list = MovieViewSet.as_view(actions={
+            "get": "list",
+            "post": "create",
+        })
+
+movie_detail = MovieViewSet.as_view(actions={
+            "get": "retrieve",
+            "put": "update",
+            "patch": "partial_update",
+            "delete": "destroy",
+        })
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("api/cinema/genres/", genre_list, name="genre-list"),
+    path("api/cinema/genres/", genre_detail, name="genre-detail"),
```

Comment:
The paths for `genre_list` and `genre_detail` are identical, which will cause a conflict. You should differentiate them, possibly by adding a parameter to `genre_detail`, such as `<int:pk>/`.